### DISCUSSION TITLE
How can I make the Terrain3D Channel Packer window bigger?

### DISCUSSION THREAD

**Discussion by lucaslevin** [TokisanGames/Terrain3D#561] [Questions] [answered]
Hello, is there any way to make this window bigger? It is impossible to do anything (I'm on Mac) .. In the earlier versions it was a bit easier, but still difficult.

<img width="1194" alt="Skærmbillede 2024-11-28 kl  22 21 43" src="https://github.com/user-attachments/assets/7722fd9f-60cb-4618-b970-eb8da3076c11">

**Comment by TokisanGames** [comment, marked as answer]
Maybe disable hdpi or display scale. This is problem is already tracked here: https://github.com/TokisanGames/Terrain3D/issues/531, and the fix was just merged.
